server.js

```diff
@@ -60,6 +60,7 @@ app.get('/logout', (req, res) => {
 })
 
 app.use("/api", require("./routes"));
+app.use("/users/user", require("./userRoutes"));
```

Comment:
There is no need for /users/user, this will create extra layers of routes. /users is sufficient.

Reply:
Yes, there was an error from my end, good observation. Will fix